knowledge-base/oracle-error-ora-01008-on-report-preview .md

```diff
@@ -0,0 +1,43 @@
+---
+title: Oracle error ORA-01008 - Not all variables bound 
+description: Oracle error ORA-01008 - Not all variables bound 
+type: troubleshooting
+page_title: Oracle error ORA-01008 - Not all variables bound 
+slug: oracle-error-ora-01008-on-report-preview 
+ticketid: 1526844
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+On previewing the report in a report viewer, an Oracle error **'ORA-01008: Not all variables are bound'** is thrown. Sometimes, the report will display without errors in the [Standalone Report Designer](../standalone-report-designer) but the error gets thrown when displaying that same report in a report viewer. 
+
+Usually, the error occurs when there is **one or more** SQL parameter/s used at **two or more places** in the query used by the [SqlDataSource component](../sqldatasource) and the data provider being used is the **Oracle.ManagedDataAccess** provider.
```

Comment:
I think the description can be a bit clearer. "one or more" is not significant for the scenario - the exception is thrown when a parameter with the same name is used more than once in the query text.